packages/myst-cli/src/frontmatter.ts

```diff
@@ -67,6 +70,34 @@ export async function getRawFrontmatterFromFile(session: ISession, file: string)
   await loadFile(session, file);
   const result = cache.$mdast[file];
   if (!result || !result.pre) return undefined;
-  const frontmatter = getFrontmatter(result.pre.mdast);
+  // Copy the mdast, this is not a processing step!
+  const frontmatter = getFrontmatter(copyNode(result.pre.mdast));
```

Comment:
@fwkoch this fixes that random bug I was seeing, where it was changing the page structure!